lua/neogit/lib/git/rebase.lua

```diff
@@ -58,6 +59,56 @@ function M.onto(start, newbase, args)
   end
 end
 
+function M.reword(commit, message, args)
+  -- based on https://peterevans.dev/posts/how-to-rewrite-git-commit-messages-non-interactively/
+  -- TODO: Error handling
+  local result = process
+    .new({
+      cmd = {
+        "git",
+        "commit",
+        "--allow-empty",
+        "--only",
+        "-m",
+        "amend! " .. commit .. "\n\n" .. message,
+      },
+      on_error = function()
+        return false
+      end,
+    })
+    :spawn_blocking()
+  if result.code ~= 0 then
+    notification.error("Rebasing failed: " .. result.stdout)
+    fire_rebase_event("conflict")
+  else
+    -- fire_rebase_event("ok")
+  end
+
+  result = process
+    .new({
+      cmd = {
+        "git",
+        "-c",
+        "sequence.editor=:",
+        "rebase",
+        "-i",
+        "--autosquash",
+        commit .. "^",
+      },
+      on_error = function()
+        return false
+      end,
+    })
+    :spawn_blocking()
```

Comment:
Same as above, but for the `-c` I think you have to pass it to `prefix()` or something like that. If you launch nvim with NEOGIT_LOG_LEVEL=trace and NEOGIT_LOG_FILE=true you can tail the log and see the exact cli strings being made. Or a well placed print() call in cli down towards the bottom where it does the string format call to build it all.

Again, on mobile, so its a bit harder to look up :)

Reply:
Simply using `cli.rebase.env({ GIT_SEQUENCE_EDITOR = ":" })....` is working for me